cmd/nfd-topology-updater/main.go

```diff
@@ -98,59 +98,62 @@ func main() {
 	if err != nil {
 		log.Fatalf("Failed to initialize NfdWorker instance: %v", err)
 	}
-		for{
```

Comment:
The PR contains a lot of indentation refactoring, making it hard to review :( 
Anyway, I will treat it as "first patch" and try to learn from it

Reply:
@marcel-apf Sorry about that Marcel, let me try to fix that to make it easier for you all to review